src/Entities/Family.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Choredo\Entities;
+
+use Ramsey\Uuid\Uuid;
+
+/**
+ * Class Family
+ * @package Choredo\Entities
+ *
+ * @ORM\Entity
+ */
+class Family
+{
+    /**
+     * @var Uuid
+     *
+     * @ORM\Id
+     * @ORM\Column(type="uuid")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $name;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", name="payment_strategy")
+     */
+    private $paymentStrategy;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer", nullable=true, name="completion_threshold")
+     */
+    private $completionThreshold;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="smallint", name="week_start_day")
+     */
+    private $weekStartDay;
+
+    /**
+     * @return Uuid
+     */
+    public function getId(): Uuid
+    {
+        if (!isset($this->id)) {
+            $this->id = Uuid::uuid4();
+        }
+        return $this->id;
+    }
+
+    /**
+     * @param Uuid $id
+     * @return Family
+     */
+    public function setId(Uuid $id): Family
+    {
+        $this->id = $id;
+
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    /**
+     * @param string $name
+     * @return Family
+     */
+    public function setName(string $name): Family
+    {
+        $this->name = $name;
+
+        return $this;
+    }
+
+    /**
+     * @return string
+     */
+    public function getPaymentStrategy(): string
+    {
+        return $this->paymentStrategy;
+    }
+
+    /**
+     * @param string $paymentStrategy
+     * @return Family
+     */
+    public function setPaymentStrategy(string $paymentStrategy): Family
+    {
+        $this->paymentStrategy = $paymentStrategy;
```

Comment:
If we're doing a setter on this there needs to be validation. I recommend Beberlei/assert